Sentry.xcodeproj/project.pbxproj

```diff
@@ -5492,21 +5499,27 @@
 				CLANG_WARN_FLOAT_CONVERSION = YES;
 				CODE_SIGN_IDENTITY = "";
 				CODE_SIGN_STYLE = Manual;
+				DEAD_CODE_STRIPPING = YES;
 				DEFINES_MODULE = YES;
 				DEVELOPMENT_TEAM = "";
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_MODULE_VERIFIER = YES;
 				ENABLE_NS_ASSERTIONS = YES;
 				ENABLE_STRICT_OBJC_MSGSEND = NO;
 				GCC_C_LANGUAGE_STANDARD = "compiler-default";
 				GCC_OPTIMIZATION_LEVEL = 0;
 				GCC_WARN_SHADOW = YES;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
 				LD_DYLIB_INSTALL_NAME = "$(DYLIB_INSTALL_NAME_BASE:standardizepath)/$(EXECUTABLE_PATH)";
+				MACOSX_DEPLOYMENT_TARGET = 11.0;
```

Comment:
We are supporting iOS 11.0 and macOS 10.13 so I guess that's why we ignored the Xcode warnings so far. I would not drop support, but maybe we should check if we can at least silence the Xcode warnings

Reply:
We have https://github.com/getsentry/sentry-cocoa/blob/main/Sources/Configuration/DeploymentTargets.xcconfig
What drove you to add the `_DEPLOYMENT_TARGET` in here?

Reply:
Xcode automatically did that in the automated perform upgrades process

Reply:
Ah OK, shouldn't we remove them?